src/static/css/pages/careers.less

```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```

Comment:
Looking into this further I found that these won't always be 150px. For instance, Working with the CFPB has them at 100x100px. I think this is fine for now, but perhaps in the future we should create two `.media_image__*` classes for the two sizes if that ends up being the only sizes we'll have for media image.